### ISSUE TITLE
why input_tensor need to minus 0.4XX 

### ISSUE THREAD

**Issue by SuperFreeMan-CXM** [sniklaus/pytorch-liteflownet#7]
Thank you code for pytorch first.
But ,  I am confused about:
		tensorFirst[:, 0, :, :] = tensorFirst[:, 0, :, :] - 0.411618
		tensorFirst[:, 1, :, :] = tensorFirst[:, 1, :, :] - 0.434631
		tensorFirst[:, 2, :, :] = tensorFirst[:, 2, :, :] - 0.454253

		tensorSecond[:, 0, :, :] = tensorSecond[:, 0, :, :] - 0.410782
		tensorSecond[:, 1, :, :] = tensorSecond[:, 1, :, :] - 0.433645
		tensorSecond[:, 2, :, :] = tensorSecond[:, 2, :, :] - 0.452793
in line 303-309.
Also,I have noticed FlowNet has similar operation:
input_transform = transforms.Compose([
        flow_transforms.ArrayToTensor(),
        transforms.Normalize(mean=[0,0,0], std=[255,255,255]),
        transforms.Normalize(mean=[0.411,0.432,0.45], std=[1,1,1])
    ])
But why?Is these operation improve the model's performance?

**Comment by sniklaus** [comment]
This is a relatively common practice that, as far as I know, originates from the early work on image recognition that leveraged convolutional neural network. For example for VGG-19, the pre-trained weights for PyTorch assume a normalization with a mean of `[ 0.485, 0.456, 0.406 ]` and a std of `[ 0.229, 0.224, 0.225 ]` as outlined in their documentation: https://pytorch.org/docs/stable/torchvision/models.html

If you want to know why LiteFlowNet uses this approach and what its effect on the quality of the estimated flow is, then I would recommend that you ask the authors directly: https://github.com/twhui/LiteFlowNet
